openshift/templates/dancer-mysql.json

```diff
@@ -330,6 +334,11 @@
       "value": "openshift/mysql-55-centos7"      
     },
     {
+      "name": "PERL_APACHE2_RELOAD",
+      "description": "Enables automatic reloading of Perl modules",
+      "value": "false"
```

Comment:
Are we actually going to compare to `true` / `false`?
What happens when the builder image sees other values like "yes", "no", "maybe"?